flexmeasures/data/schemas/reporting/aggregation.py

```diff
@@ -0,0 +1,67 @@
+from marshmallow import fields, ValidationError, validates_schema
+
+from flexmeasures.data.schemas.reporting import ReporterConfigSchema
+
+from enum import Enum
+
+
+class AggregationMethod(Enum):
+    SUM = "sum"
+    MEAN = "mean"
```

Comment:
For your consideration: if we remove this `Enum` altogether, no code changes would be necessary on our side to expand the list of supported aggregation methods as they are (or become) available in Pandas. On the other hand, it might lead to unforeseen / unintended use cases.

I already tried expanding this `Enum` with "median", "std", "min", "max" and "prod", which all seem useful to me in practice. (I'm hesitant about "prod", though, because it reflects a change of unit.)

Expected test values, respectively: 1, 0, 1, 1 and 1 (maybe the test data should be set up to be a little more interesting).